docs/bigquery-usage.rst

```diff
@@ -34,31 +34,74 @@ Authorization / Configuration
      >>> client = bigquery.Client()
 
 - Override the credentials inferred from the environment by passing explicit
-  ``credentials`` to one of the alternative classmethods, `from_service_account_json`:
+  ``credentials`` to one of the alternative ``classmethod`` factories,
+  `:meth:gcloud.bigquery.client.Client.from_service_account_json`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_json('/path/to/creds.json')
 
-  or `from_service_account_p12`:
+  or `:meth:gcloud.bigquery.client.Client.from_service_account_p12`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_p12('/path/to/creds.p12', 'jrandom@example.com')
 
-- Override the project inferred from the environment by passing an explicit
-  ``project`` to the constructor, or to either of the alternative classmethods:
+
+Projects
```

Comment:
Well that would make sense -- and the "list projects" then is really RM API's "list projects" filtered by "has BQ enabled". If BQ gets clicked on by default this will match up. I'm still going to make the argument that this is a "doesn't belong here" (as evidenced by the fact that all other APIs don't have this) and that it's only here because it existed before GCP existed... 

Any compromise that makes everyone happy and doesn't require us to potentially confuse people with docs that make you wonder "wait... wat?"

Maybe we privatize / don't publicly document the method? If someone is dying for listing BQ enabled projects, they have the ability to do it? But it doesn't get in people's faces with explanations of what a project is and all that?